x-pack/plugins/observability_solution/logs_explorer/public/state_machines/data_views/src/defaults.ts

```diff
@@ -13,7 +13,7 @@ export const createDefaultContext = (): DefaultDataViewsContext => ({
   dataViewsSource: null,
   dataViews: null,
   error: null,
-  search: {
+  searchCriteria: {
```

Comment:
I might explained it wrong, when I talked about searchCriteria it was not meant to rename the `search` property in the state machine context. My suggestion was about refactoring only the `searchDataViews` function to receive a parameter object as a second argument, but still using it with separate context properties:
```ts
searchDataViews(views, {search: context.search, filter: context.filter})
```
I'd rather keep them separate in the context and keep the `search` naming for consistency with integrations and datasets state machines.

Reply:
Gotcha, updated